tests/Doctrine/Tests/DBAL/Functional/Schema/SchemaManagerFunctionalTestCase.php

```diff
@@ -51,10 +52,19 @@ public function testDropsDatabaseWithActiveConnections()
 
         $this->_sm->dropAndCreateDatabase('test_drop_database');
 
-        $this->assertContains('test_drop_database', $this->_sm->listDatabases());
+        $knownDatabases = $this->_sm->listDatabases();
+        if ($this->_conn->getDatabasePlatform() instanceof OraclePlatform) {
```

Comment:
This indicates a separate issue inside the OraclePlatform IMO. Shouldn't the uppercase conversion happen in there, if upper case is enforced?